posts/2021-11-22-til-cleaner-queries-with-ecto-map.md

```diff
@@ -13,7 +13,7 @@ Today I learned how to write cleaner `Ecto` select queries with the help of `Ect
 
 # TIL: Cleaner queries with Ecto `map`
 
-When working with large tables, is a common practice to avoid using `SELECT *` to make a beter use of computer resources and database indices.
+When working with large tables, a common practice avoids using `SELECT *` to make better use of computer resources and database indices.
```

Comment:
Thanks @angrycandy! I think we may still be missing something from this sentence 🤔 How does this read:

```suggestion
When working with large tables a common practice is to avoid using `SELECT *` to make better use of database indices and resources.
```

I also rearranged the end of the sentence to emphasize the benefit is to indices. 

cc @SophieDeBenedetto